src/schemas.py

```diff
@@ -181,4 +180,4 @@ class MaxCallsPerEntrypointCondition(ConditionBase):
 
 
 class MaxCallsPerSponseeCondition(ConditionBase):
-    sponsee_address: str
+    pass
```

Comment:
Who does this condition apply to?

Reply:
It was called from this function [create_max_calls_per_sponsee_condition](https://github.com/marigold-dev/gas-station/blob/quyen%40max_sponsee/src/crud.py#L350)